media/com_finder/js/index.es6.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+Joomla = window.Joomla || {};
+
+(() => {
+  'use strict';
+
+  document.addEventListener('DOMContentLoaded', () => {
+    Joomla.submitbutton = (pressbutton) => {
+      if (pressbutton === 'index.purge') {
+        if (window.confirm(Joomla.JText._('COM_FINDER_INDEX_CONFIRM_PURGE_PROMPT'))) {
```

Comment:
What should we do with the message <tt>Unexpected confirm no-alert</tt>.
Should we use a custom alert or should we use <tt>/* eslint-disable no-alert */</tt>.
Can we use <tt>joomla-alert</tt> for this?

Reply:
<blockquote>What should we do with the message Unexpected confirm no-alert.</blockquote>
Leave as is it. We need a custom element for this.<br>


<blockquote>Can we use joomla-alert for this?</blockquote>
<code>Joomla-alert</code>  is only for message rendering. Not for <code>confirm</code> form alert.

@C-Lodder right!

Reply:
Since we don't have a usable substitution for confirm (yet) I'll propose to add a comment with a TODO replace with joomla-alert
and once we get the newer version of alerts that will support confirm we just replace this line with the appropriate code

Reply:
Would a usable substitution be placed in the file <tt>/media/vendor/joomla-custom-elements/js/joomla-alert.js</tt>?
Should we do it in this repo?

Reply:
No the repo for the ui components is not the same as the main joomla repo

Reply:
Agree: please add a //TODO replace with joomla-alert and then we can merge this.